### PR TITLE
necessary packages for python images

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

File: python/nitrogen6x/jessie/Dockerfile [hunk 1]
```diff
@@ -5,6 +5,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 		python-dev \
 		python-dbus \
 		python-pip \
+		python-virtualenv \
+		python-setuptools \
 	&& rm -rf /var/lib/apt/lists/*
```
Comment (emirotin) [inline python/nitrogen6x/jessie/Dockerfile]: are all of these files generated from the template file?

**Comment by emirotin** [comment]
Assuming the answer yes LGTM

**Comment by petrosagg** [comment]
@emirotin yes. For these type of PRs it's better to check commit by commit as there is one that has the template changes and then another one with all the autogenerated files from the templates.

**Comment by petrosagg** [comment]
SGTM

**Pull request merged**